GOALS.md

```diff
@@ -3,13 +3,13 @@
 
 The ultimate goal is to make the development of apps adhering to [WAI-ARIA](http://www.w3.org/TR/wai-aria/) a breeze. The ultimatest™ goal is to make this library obsolete by lobbying the W3C and browser vendors into supporting the basic tools provided by ally.js. (One may dream…)
 
-Applications have to deal with quite a number of different issues to get to a state we may call "accessible". There's the issue of *maintaining the focus*, a subset of providing *keyboard navigation*. Then there's [WAI-ARIA](http://www.w3.org/TR/wai-aria/) to provide help the browser provide the operating system's Accessibility Infrastructure with the necessary details.
+Applications have to deal with quite a number of different issues to get to a state we may call "accessible." There's the issue of *maintaining the focus*, a subset of providing *keyboard navigation*. Then there's [WAI-ARIA](http://www.w3.org/TR/wai-aria/), which helps the browser provide the operating system's Accessibility Infrastructure with the necessary details.
```

Comment:
Per English style guides (MLA, APA, and Chicago), period, comma, question mark, exclamation marks go of inside quotation marks. But colons and semicolons go outside of quotation marks.

American English is funny that way.

<hr>

_Sources:_
- https://owl.english.purdue.edu/owl/resource/577/03/
- http://blog.apastyle.org/apastyle/2011/08/punctuating-around-quotation-marks.html
 http://www.chicagomanualofstyle.org/qanda/data/faq/topics/Punctuation.html
- http://www.grammarly.com/answers/questions/4726-quotation-marks-and-other-punctuation-inside-or-outside/